index.html

```diff
@@ -37,9 +37,9 @@
     <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
   </body>
 </html>
-<script src="scripts/jquery.js"></script>
-<script src="scripts/popup.js"></script>
-<script src="scripts/bootstrap.min.js"></script>
+
+<script type="text/javascript" src="scripts/popup.js"></script>
+<script type="text/javascript" src="scripts/bootstrap.min.js"></script>
```

Comment:
@rgpt  thanks for taking the time to go through my code and reviewing it. As for your review, I have already seen the error and it is a custom exception which is thrown if jquery is not there. Even if it is being shown on the console, it doesn't have any effect on speed or efficiency of extension.